TypeCobol.Test/Parser/Programs/TypeCobol/ProcedureInput.rdzPGM.txt

```diff
@@ -0,0 +1,33 @@
+--- Diagnostics ---
```

Comment:
Can you use a "Mix" file rather than "PGM" ?

I mean, the file should be named `ProcedureInput.rdzMix.txt`
It'll be easier to check diagnostics with a "Mix" file.